ivy_tests/test_ivy/test_frontends/test_paddle/test_tensor/test_tensor.py

```diff
@@ -2401,6 +2445,7 @@ def test_paddle_cholesky(
     )
 
 
+# clip
 @handle_frontend_method(
```

Comment:
please remove these comments unrelated to your current PR.

Reply:
Addressed